### PR TITLE
RHELPLAN-41563 - Logging - splitting an input/output config file into…

### PR DESCRIPTION
**Pull request by nhosoi**
… multiple ones that contains just one operation 2

Sharing one input_template.j2 between the basics and files inputs.

### CODE REVIEW

File: roles/rsyslog/tasks/inputs/basics/main.yml [hunk 1]
```diff
@@ -9,9 +9,12 @@
 
 - name: Create basics input configuration files in /etc/rsyslog.d
   vars:
+    __line_0: ""
```
Comment (richm) [inline roles/rsyslog/tasks/inputs/basics/main.yml:12]: please rename this to something like `__rsyslog_input_decl`

File: roles/rsyslog/tasks/inputs/basics/main.yml [hunk 2]
```diff
@@ -9,9 +9,12 @@
...
+    __my_name: "{{ basics_item.name }}"
+    __my_type: "{{ basics_item.type }}"
```
Comment (richm) [inline roles/rsyslog/tasks/inputs/basics/main.yml:14]: please use `__rsyslog_input_name` and `__rsyslog_input_type`

**Comment by richm** [comment]
other than a couple of minor nits, looks good

**Comment by nhosoi** [comment]
Thanks for your reviews and advice, @richm.  Merging the PR...

**Pull request merged**